lib/transports/__es__/_helpers.js

```diff
@@ -209,6 +209,8 @@ const closePit = async (self) => {
   if (!self.pitId) return
 
   const closeRequest = {
+    // FIXME: OS got different URL and body
+    // https://docs.opensearch.org/docs/latest/search-plugins/searching-data/point-in-time-api/#delete-pits
```

Comment:
self has context. you can detect openSearchServerless and then change the url if true.